TbsCore/Helpers/ProxyHelper.cs

```diff
@@ -94,27 +95,39 @@ private static void CreateFile(string path, string text)
         public static async Task TestProxies(List<Access> access)
         {
             List<Task> tasks = new List<Task>(access.Count);
+            var restClient = new RestClient("https://api.ipify.org/");
             access.ForEach(a =>
             {
-                tasks.Add(Task.Run(() =>
+                tasks.Add(Task.Run(async () =>
                 {
-                    var restClient = HttpHelper.InitRestClient(a, "https://api.ipify.org/");
-                    a.Ok = ProxyHelper.TestProxy(restClient, a.Proxy);
+                    // Set proxy

```

Comment:
Isn't this same exect functionality inside **InitRestClient**? Don't Repeat Yourself:)

Reply:
InitRestClient make another restclient, this use same restclient but change proxy variable